src/main/resources/standardTestCases/multiline_command_line.test

```diff
@@ -0,0 +1,13 @@
+name: multiline_command_line
+testFormat: workflowsuccess
+ignore: true # Fails on JES, Succeeds locally. WTF?
```

Comment:
I'm glad you pointed this out tho, I'd forgotten about it and investigating what's going on here is perhaps the most important thing to come out of this PR